src/core/main.cpp

```diff
@@ -636,15 +666,13 @@ int main( int argc, char * * argv )
 		Engine::init( true );
 		destroyEngine = true;
 
-		QFileInfo fileInfo( fileToLoad );
-		if ( !fileInfo.exists() )
-		{
-			printf("The file %s does not exist!\n", fileToLoad.toStdString().c_str());
-			exit( 1 );
-		}
-
 		printf( "Loading project...\n" );
 		Engine::getSong()->loadProject( fileToLoad );
+		if( Engine::getSong()->isEmpty() )
+		{
+			printf("The project %s is empty, aborting!\n", fileToLoad.toUtf8().constData() );
+			exit( EXIT_FAILURE );
+		}
```

Comment:
oK. I just skimmed past some comment on the net that exit() could be a problem with multi threaded applications. I'm happy with .lmmsrc.xml not being written so we keep failing loads from appearing in the recently opened projects list.